package.json

```diff
@@ -20,12 +20,16 @@
   },
   "homepage": "https://github.com/socialtables/react-infinity-menu",
   "dependencies": {
+    "lodash": "^4.16.2",
     "nested-objects": "0.0.2"
   },
   "devDependencies": {
     "babel": "^5.8.23",
     "babel-core": "^5.8.25",
-    "babel-loader": "^5.3.2",
+    "babel-loader": "^5.4.2",
```

Comment:
if we want to upgrade the babel-loader, why not just updated to the latest version?